probe/tag/pidtree.go

```diff
@@ -24,7 +24,7 @@ var (
 	readFile = ioutil.ReadFile
 )
 
-func newPIDTree(procRoot string) (*pidTree, error) {
+func newRealPIDTree(procRoot string) (*pidTree, error) {
```

Comment:
Creating a local reference to a standard function/constructor to swap out for tests is a common enough idiom, easily identifiable with some exposure. I've never seen Stub or Ref or anything in the name of the reference var, i.e.

```go
var (
    now = time.Now     // this
    nowStub = time.Now // not this
)
```

And I'd argue it's more awkward to parse in the place where it matters the most,

```go
func doSomething() {
    object := newFooStub() // stub? why? isn't this the real code?
    object := newFoo()     // makes more sense...
}
```

If you're still not convinced, I defer.